res/css/views/rooms/_RoomHeader.scss

```diff
@@ -141,17 +141,25 @@ limitations under the License.
 }
 
 .mx_RoomHeader_topic {
+    $line-height: 1.2em;
+    $lines: 2;
+
     flex: 1;
     color: $roomtopic-color;
     font-weight: 400;
     font-size: $font-13px;
+    border-bottom: 1px solid transparent;
+    line-height: $line-height;
+    max-height: calc($line-height * $lines);
+
     // to align baseline of topic with room name
     margin: 4px 7px 0;
+
     overflow: hidden;
-    text-overflow: ellipsis;
-    border-bottom: 1px solid transparent;
-    line-height: 1.2em;
-    max-height: 2.4em;
+    line-clamp: $lines; // https://drafts.csswg.org/css-overflow-3/#line-clamp
```

Comment:
It says:
> For compatibility with legacy content, UAs that support line-clamp must also support the -webkit-line-clamp property ...

> Implementations of the legacy -webkit-line-clamp property have not behaved identically to what is specified here. ...

But since this is the draft, the documentation is not finalized or standardized, so the fact is that `-webkit-line-clamp` is not actually legacy yet?

Anyway `-webkit-line-clamp ` will be kept available per the document, so I'll remove this for now.